.github/workflows/test-hygiene.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           repository: ${{ needs.changes.outputs.mergeable == 'true' &&  github.repository || needs.changes.outputs.source_repo }}
       - name: Generate Notebooks
         run: |
-          pip install nbformat && cd demo && python generate_notebooks.py
+          pip install nbformat && cd python scripts/generate_notebooks.py
```

Comment:
typo: remove cd

Reply:
I think typo still needs to be fixed @abidlabs

Reply:
😱

Reply:
fixed now